tools/libcloud/gcp.py

```diff
@@ -189,35 +195,35 @@ def wait_boot_node(gce):
     raise RuntimeError("boot node did not start in 600 seconds")
 
 
-def count_core_nodes(gce, region):
+def count_core_nodes(gce, region, prefix_txt="cores-"):
```

Comment:
Could this variable simply be called `prefix`?

Reply:
changed